icondbtools/libs/block_database_raw_reader.py

```diff
@@ -17,6 +39,27 @@ def close(self):
             self._db.close()
             self._db = None
 
+    @property
+    def db(self):
+        return self._db
+
+    def get_transaction_by_hash(self, tx_hash: bytes) -> bytes:
+        transaction: bytes = self._db.get(tx_hash)
+        return transaction
+
+    @staticmethod
+    def get_transactions_from_block(block: bytes) -> list:
+        block: dict = json.loads(block)
+        version: str = block["version"]
+        if version == "0.1a":
+            transactions = block["confirmed_transaction_list"]
+        elif version == "0.3" or version == "0.4":
+            transactions = block["transactions"]
```

Comment:
Version check is too strict.
I think that keyword "transactions" will not be changed although block version is changed later.